lib/main_common.pm

```diff
@@ -2371,15 +2371,18 @@ sub load_virtualization_tests {
     if (get_var("STANDALONEVT")) {
         loadtest "virtualization/prepare";
     }
+    # FIXME: change order back
```

Comment:
for testing you can also schedule tests with `EXCLUDE_MODULES=virtualization/yast_virtualization,virtualization/virt_install` etc. or simply create another dedicated test scenario for vagrant only